
package fr.urssaf.image.sae.ihm.web.exploit.test.service.impl;

import java.util.List;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.springframework.test.context.ContextConfiguration;
import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;


import fr.urssaf.image.sae.ihm.web.exploit.exception.ErreurTechniqueException;
import fr.urssaf.image.sae.ihm.web.exploit.modele.SaeResultatRecherche;
import fr.urssaf.image.sae.ihm.web.exploit.service.RechercherService;
import fr.urssaf.image.sae.ihm.web.exploit.service.impl.RechercherServiceImpl;

@RunWith(SpringJUnit4ClassRunner.class)
@ContextConfiguration(locations = "/applicationContext_test.xml")
public class RechercherServiceImplTest {
   
   @Test(expected = IllegalArgumentException.class)
   public void testRechercherDocument() {
      String requeteLucene = null;
      List<String> codes = null;

      RechercherService rs = new RechercherServiceImpl();
      try {
         SaeResultatRecherche resRecherche = rs.rechercherDocuments(requeteLucene, codes);
      } catch (ErreurTechniqueException e) {
         // TODO Auto-generated catch block
         e.printStackTrace();
      }
            
   }
   
}
